Marlin/src/module/configuration_store.cpp

```diff
@@ -422,9 +423,13 @@ void MarlinSettings::postprocess() {
       EEPROM_WRITE(storage_slot);
     #endif // AUTO_BED_LEVELING_UBL
 
-    // 10 floats for DELTA / [XYZ]_DUAL_ENDSTOPS
+    // 11 floats for DELTA / [XYZ]_DUAL_ENDSTOPS
     #if ENABLED(DELTA)
       EEPROM_WRITE(delta_height);              // 1 float
+      #if DISABLED(DELTA_AUTO_CALIBRATION)
+        float raw_delta_height;
```

Comment:
`raw_delta_height` needs to be initialized to 0.0

Reply:
ok